code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm

```diff
@@ -69,9 +69,9 @@
 		if(prob(reac_volume))
 			W.ScrapeAway()
 
-/datum/reagent/clf3/expose_mob(mob/living/M, method=TOUCH, reac_volume)
+/datum/reagent/clf3/expose_mob(mob/living/M, methods=TOUCH, reac_volume)
 	if(istype(M))
-		if(method != INGEST && method != INJECT)
+		if(methods & ~(INGEST|INJECT))
```

Comment:
```suggestion
		if(!(methods & (INGEST|INJECT)))
```

Reply:
I think this intends for the fire stacks and other effects to be applied so long as the ClF3 contacts the exterior of the mob. The way you are suggesting would prevent any exposure involving (TOUCH|PATCH|VAPOR) from doing so if the methods also included (INGEST|INJECT).

Reply:
Ah, if you want this to succeed in the case of `NONE` and `TOUCH|INGEST` then your method is better.